subsys/dfu/boot/mcuboot.c

```diff
@@ -34,8 +34,26 @@
 #define BOOT_HEADER_MAGIC_V1 0x96f3b83d
 #define BOOT_HEADER_SIZE_V1 32
 
+#if USE_PARTITION_MANAGER
+
```

Comment:
I think we may also need this one: `#include <pm_config.h>`

Reply:
but yes, will apply your request